guides_and_processes/engineering_meetings.md

```diff
@@ -0,0 +1,237 @@
+- Celebrate the wins.
+
+**What Not to Do:**  
+- Avoid blaming each other; focus on processes and systems.
+- Do not delve into detailed technical design debates; keep it focused on team efficiency and workflows.
+
+**Who Should Attend:**  
+- **Mandatory:** Architects, Engineering Manager, Backend & Frontend Developers, Product Owner, Designers, Design Manager, Testers/QA
+- **Recommended:** SRE, Operations (if involved in sprint tasks)
+
+**Inputs / Preparation:**  
+- Think about what worked well and what could be better.
+- Come prepared with constructive suggestions for improvement.
+
+**Expected Outputs:**  
+- A set of action items to improve team processes and collaboration.
+- Documented lessons learned to carry forward.
+- Action items with an owner to complete any follow up required.
+
+**Frequency:**  
+- End of each sprint.
+
+---
+
+### Sprint Planning (Start of Sprint)
+
+**Purpose:**  
+- Select, clarify, and commit to a set of “Ready for Dev” items for the upcoming sprint.
+
+**What to Do:**  
+- Define high level goal for the sprint.
+- Review prioritized backlog items that meet the “Ready” criteria.
+- Confirm estimates, acceptance criteria, and dependencies.
+- Ensure team capacity aligns with the planned work.
+
+**What Not to Do:**  
+- Do not introduce new, unrefined items on the spot.
+- Do not create new tickets, tickets at this stage should have been viewed by many people and be in an acceptable state for engineers.
+- Avoid deep architectural debates—if needed, schedule a separate tech discussion if required.
+
+**Who Should Attend:**  
+- **Mandatory:** Architects, Engineering Manager, Backend & Frontend Developers, Product Owner  
+- **Recommended:** Designers, Design Manager (if design tasks are included), Testers/QA (if testing tasks are included)
+  
+**Inputs / Preparation:**  
+- Reviewed and prioritized backlog items.
+- Acceptance criteria and estimates in place. (can be completed by engineers separate from meetings)
+- Any known dependencies identified.
+
+**Expected Outputs:**  
+- A sprint backlog with clearly defined tasks ready to be worked on in the next spring.
+- A shared understanding of the sprint goals and commitments.
+
+**Frequency:**  
+- Start of each sprint (e.g., every 2 weeks).
+
+---
+
+### Backlog Grooming
+
+**Purpose:**  
+- Review candidate tickets/epics that have been prepared to ensure they have clear acceptance criteria, defined scope, and no outstanding dependencies.  
+- Discuss estimates and confirm if the size of each task is manageable within a single sprint.  
+- Validate that each item aligns with overall product and architectural goals.  
+- Prioritize or re-prioritize items based on current project needs and constraints.  
+- Identify any items needing further refinement before they can be moved into a “Ready for Dev” state.
+
+**What to Do:**  
+- Review candidate tickets/epics, clarify acceptance criteria, confirm estimates.
```

Comment:
Does it mean that [Architects, Engineering Manager, Product Owner] will estimate tickets? If not then devs, or sub-teams, should be involved at some points in this process

Reply:
No estimation is a team effort, the engineers who will be doing the implementation are much better positioned to do the estimation, Architects, Engineering Manager, Product Owner should be used for clarification of any points which are ambiguous or unclear for the purposes of estimation, let me take a look at the wording again to see if that is unclear.

Reply:
Changed wording